src/openrct2/actions/GameAction.h

```diff
@@ -318,6 +319,18 @@ struct GameAction
         return 0;
     }
 
+    int32_t NormaliseTrackType(int32_t _trackType) const
```

Comment:
This seems like the wrong place to put it in, this has hardly anything to do with game actions in general. It would be better suited with the TrackRemoveAction class.

Reply:
We also have `TrackElement::IsStation()` and `track_type_is_station` which could be used.
E.g.
```
if (track_type_is_station(_trackType)
    _trackType = TrackElemType::EndStation;
```